docs/about-filecoin/ipfs-and-filecoin.md

```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
 
-## Incentivizing data reliability
+This page aims to explain the relationship between the IPFS and Filecoin and assist users in deciding which option, or combination of options, is best suited for their use-case.
 
-[IPFS](https://ipfs.io) is a free and open-source network protocol that allows users to store and transfer verifiable data with each other. IPFS users persist data on the network by [pinning](https://docs-beta.ipfs.io/concepts/persistence/#pinning-in-context) it on their own hardware, to a third-party pinning service, or through groups of individual IPFS users sharing resources to ensure the content remains live.
+## Data storage incentives
 
-The lack of a built-in incentive mechanism is the challenge Filecoin hopes to solve by allowing users to incentivize long-term distributed storage at competitive prices through a marketplace of storage nodes while maintaining the efficiency and resiliency provided by the IPFS network.
+[IPFS](https://ipfs.io) allows users to store and transfer verifiable, content-addressed data in a peer-to-peer network. IPFS users persist the data they want **on their own IPFS nodes**. This is referred as [pinning](https://docs.ipfs.io/concepts/persistence). Sometimes the data may be pinned using a third-party pinning service, or through groups of individual IPFS users. The data exists in the network as long as one user is storing it and able to provide it to others when they request it.
 
-## Different persistence strategies: which should I use?
+However, IPFS does not include a built-in mechanism to incentivize the storage of data for _other_ people. This is the challenge Filecoin hopes to solve: the Filecoin network creates a distributed storage market place for long-term storage where nodes with large storage capacity can rent it to users with data-storage needs, and get paid for it.
 
-Depending on your use case, one or multiple of these persistence strategies might be a good fit for you.
+The cryptographic guarantees ensuring that data is safely stored by the Filecoin network make the storage and retrieval of data a computationally and time-expensive operation. For this reason, Filecoin enables an additional retrieval market where dedicated nodes can help quickly delivering content from the network for a payment. This delivery mechanism may make use of IPFS.
 
-### Using IPFS
+In that sense, Filecoin can be seen as a "cold" storage layer, perfect to safely store large batches of data, while IPFS would be the "hot" storage layer, designed for the quick retrieval and distribution of content.
+
+## Which should I use?
```

Comment:
Better systems. They are not single protocols. Protocol suites, if anything, and still it is complex because of how loaded the word "protocol" is in network architectures. Therefore I try to stay away from it.